docs/usermanual/component_reference.html

```diff
@@ -3813,7 +3813,7 @@ <h3 id="JMS_Publisher_parms1">
   
```

Comment:
HTML files are generated from XML. Could you update comonent_reference.xml instead ?